ops/cmd/create_config/main.go

```diff
@@ -48,12 +56,22 @@ func action(cliCtx *cli.Context) error {
 	}
 
 	statePath := cliCtx.String(StateFilename.Name)
+	opDeployerBinDir := cliCtx.String(OpDeployerBinDir.Name)
 
-	err = manage.GenerateChainArtifacts(statePath, wd, cliCtx.String(Shortname.Name), nil, nil, 0, "")
+	err = manage.GenerateChainArtifacts(statePath, wd, cliCtx.String(Shortname.Name), nil, nil, 0, "", opDeployerBinDir)
 	if err != nil {
 		return fmt.Errorf("failed to generate chain config: %w", err)
 	}
 
 	output.WriteOK("done")
 	return nil
 }
+
+func defaultBinDir() string {
```

Comment:
There's some code duplication here with the import_devnet package, but I'm OK with it for the time being.